bin/alluxio

```diff
@@ -160,8 +160,13 @@ function runJavaClass {
   for arg in "$@"; do
       shift
       case "${arg}" in
-          -D*) ALLUXIO_SHELL_JAVA_OPTS+=" ${arg}" ;;
-          *) CLASS_ARGS+=" ${arg}"
+          -debug)
+              echo "debug mode!"
+              ALLUXIO_USER_JAVA_OPTS+=" ${ALLUXIO_SHELL_DEBUG_JAVA_OPTS}" ;;
```

Comment:
either do `ALLUXIO_SHELL_JAVA_OPTS+=" ${ALLUXIO_SHELL_DEBUG_JAVA_OPTS}"` or `ALLUXIO_USER_JAVA_OPTS+=" ${ALLUXIO_USER_DEBUG_JAVA_OPTS}"` ... 

the latter actually makes more sense as this can be use with `bin/alluxio runClass` as well